README_.md

```diff
@@ -0,0 +1,177 @@
+# 1 BladeDISC
+
+BladeDISC is an end-to-end **DynamIc Shape Compiler** project for machine learning
+workloads, which is one of the key components of the Alibaba's [PAI-Blade](https://www.aliyun.com/activity/bigdata/blade).
+BladeDISC provides general, transparent and ease of use performance optimization
+for TensorFlow/PyTorch workloads on GPGPU and CPU backends. The architecture 
+natively supports dynamic shape workloads, with many considerations in the 
+performance of both static and dynamic shape scenarios. It also supports multiple
+and flexible deployment solutions, including both Plugin Mode inside
+TensorFlow/Pytorch runtime, and Standalone Mode for AOT standalone execution.
+The project is based on [MLIR](https://mlir.llvm.org/) and highly related with
+[mlir-hlo](https://github.com/tensorflow/mlir-hlo) project.
+
+Refer to [our website](https://alibaba.github.io/BladeDISC/) for more
+information, including the setup tutorial, developer guide, demo examples and
+documents for developers.
+
+## 1.1 Features and Roadmap
+
+### 1.1.1 Frontend Framework Support Matrix
+
+|           | TensorFlow [1] | Pytorch [2]  |
+|---------- | -------------- | ------------ |
+| Inference |    Yes         |    Yes       |
+|  Training |    Yes [3]     |  Ongoing     |
+
+[1] TensorFlow ver1.12, 1.15, 2.4 & 2.5 are supported and fully verified. For
```

Comment:
> `ver1.12 `

`ver` is not Tensorflow version prefix, maybe `1.12` is fine.

Reply:
done